Python/ast.c

```diff
@@ -2780,6 +2852,26 @@ ast_for_call(struct compiling *c, const node *n, expr_ty func, bool allowgen)
                     return NULL;
                 asdl_seq_SET(args, nargs++, e);
             }
+            else if (TYPE(CHILD(ch, 1)) == COLONEQUAL) {
+                /* treat colon equal as positional argument */
+                if (nkeywords) {
```

Comment:
I prefer to avoid code duplication. Maybe merge this case with the case for `NCH(ch) == 1`?
```c
if (NCH(ch) == 1 || (NCH(ch) == 3 && TYPE(CHILD(ch, 1)) == COLONEQUAL)) {
    if (nkeywords) {
        ...
    }
}
```